src/assets/properties-panel.css

```diff
@@ -537,8 +544,9 @@ select.bio-properties-panel-input {
 }
 
 .bio-properties-panel-input[type="checkbox"]:focus {
-  outline: 2px solid var(--input-focus-border-color);
-  outline-offset: 0;
+  outline: auto;
```

Comment:
Why `auto`? It let's the user agent to render a custom outline style. Is this what we want?

Reply:
Then I'd expect it to be consistent. If we already set the color we should also set the style. As you said, one can always override.

Reply:
I agree. We should maintain consistent styling across all child libraries instead of relying entirely on integrators. While we can allow them the flexibility to override styles, our defaults should not depend on their configurations.

Reply:
We've already specified the color in the same class. Using the auto property ensures an default outline is provided, overriding the none outline for inputs.

```javascript
.bio-properties-panel-input:focus,
.bio-properties-panel-input:focus-within {
  outline: none;
```